posthog/tasks/test/test_update_cache.py

```diff
@@ -91,6 +92,12 @@ def _create_dashboard_tile_with_known_cache_key(
     return dashboard, tile
 
 
+class TestTurboCacheUpdate(APIBaseTest):
+    @skip("how to patch get_client???")
```

Comment:
`@patch("posthog.tasks.update_cache.get_client")` not working?

Reply:
I switched and wrote a different test... Couldn't get a patch to be picked up inside the `update_cached_item_task` task.

Since it isn't a straight passthrough anymore the correct route is probably to delegate through to a new method in the `update_cache` module where something (probably Celery) isn't getting in the way of the patch